tests/client/amo/components/TestAddonDetail.js

```diff
@@ -25,9 +27,11 @@ function render({ addon = fakeAddon, ...customProps } = {}) {
   };
 
   return findRenderedComponentWithType(renderIntoDocument(
-    <I18nProvider i18n={i18n}>
-      <AddonDetail {...props} />
-    </I18nProvider>
+    <Provider store={createStore(initialState)}>
```

Comment:
I've addressed this type of thing by passing in a stub `AddonMoreInfo` component to isolate it from the test (it has its own tests anyway). Adding a `Provider` doesn't seem so bad in this case though.